### ISSUE TITLE
bridge.js doesn't support NodeJS

### ISSUE THREAD

**Issue by desunit** [bridgedotnet/Bridge#1943]
I've noticed that Bridge.js doesn't support NodeJS require. Once you try to import it to simple NodeJS app:

```
app.js:
b = require('./bridge.js');

with command line:
node app.js

you will get the output:
bridge.js:1463
    System.Nullable = nullable;
    ^

ReferenceError: System is not defined
    at /Users/test/Documents/Visual Studio 2015/Projects/ClassLibrary1/ClassLibrary1/Bridge/output/bridge.js:1463:5
    at Object.<anonymous> (/Users/test/Documents/Visual Studio 2015/Projects/ClassLibrary1/ClassLibrary1/Bridge/output/bridge.js:23033:3)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/test/Documents/Visual Studio 2015/Projects/ClassLibrary1/ClassLibrary1/Bridge/output/app.js:1:67)
```

Any plans to support NodeJS it in future?

**Comment by desunit** [comment]
It look I've just found a workaround. The cause is in the scope of Init:
```
(function (globals) {
...
globals.System = {}

and latter in the code System is used without "globals" scope.

```

I replaced globals with "this" scope for nodejs:

```
(function (globals) {
    if (typeof module !== "undefined" && module.exports) {
        globals = this;
        navigator = {};
        navigator.userAgent = "";
    }

    // @source Core.js

    var core = {
        global: globals,
```

not sure if it's great solution - I can send a pull request if it's ok for you.

**Comment by LeonidVeriga** [comment]
Hi @desunit,

Thanks for reporting the issue and a workaround.
We will try to fix in the upcoming 15.3 release.
